### ISSUE TITLE
Add "paper-checkbox-checked-background-color" attribute

### ISSUE THREAD

**Issue by eximius313** [PolymerElements/paper-checkbox#91]
In paper-checkbox, one can style pair: ```unchecked-background-color``` and ```unchecked-color```
Unfortunately for ```checked``` style there is only ```checked-color``` attribute.
Could you please add ```--paper-checkbox-checked-background-color``` style attribute in order to complement the functionality of styling?

Thank you in advance

**Comment by notwaldorf** [comment]
Isn't that what `--paper-checkbox-checked-color` does? If you look at the last demo in https://elements.polymer-project.org/elements/paper-checkbox?view=demo:demo/index.html&active=paper-checkbox, the background colors change for both checkboxes.

**Comment by eximius313** [comment]
No. I need colors to be inversed. White background with colored check.

**Comment by skcorroh** [comment]
Why can you control unchecked background/border separately, but checked is forced to be the same? I need to control them separately as well
![image](https://user-images.githubusercontent.com/21206433/41185829-cf46bb12-6b49-11e8-8f29-3349b9946ee3.png)
